app/src/main/java/net/squanchy/proximity/preconditions/ModularProximityPreconditions.java

```diff
@@ -46,6 +47,7 @@ public boolean onRequestPermissionsResult(int requestCode, String[] permissions,
         }
     }
 
+    @SuppressWarnings("PMD.UseVarargs") // This array is passed to the Android APIs and is kept for consistency
```

Comment:
It really looks weird to have it here and not on the other method which takes two consecutive `int[]`, as written in the comment I preferred consistency between the methods and with the system API they map to. I would not change it to a varargs at all _in this case_